kubespawner/spawner.py

```diff
@@ -150,7 +150,7 @@ def _namespace_default(self):
     ).tag(config=True)
 
     pod_name_template = Unicode(
-        'jupyter-{username}',
+        'jupyter-{username}' + '-{servername}' if getattr(self, 'name', None) is not None,
```

Comment:
I think this can be '{username}{servername}' and add the `-` *in* `servername` when the template is applied.